### PR TITLE
fix: [PL-42961]: add validation on pasting string with over 128 characters

### PR DESCRIPTION
**Pull request by sahibaHarness**
### Summary

add validation on pasting string with over 128 characters
#### Screenshots

https://github.com/harness/harness-core-ui/assets/112424209/2bb38fa2-6515-4bad-ab7f-5bb077f3c229

#### PR Checklist

<details>

https://github.com/harness/harness-core-ui/assets/112424209/5941d547-416c-4ca4-a38a-b782fb849849

<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
- MessageMetadata: `trigger messagecheck`
- GitLeaks: `trigger gitLeaks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

### CODE REVIEW

**Review by tiwaritejas — APPROVED** [review]

**Review by Sagar-Rajput7 — APPROVED** [review]

**Review by reetika04 — APPROVED** [review]

**Review by adiyaar24 — APPROVED** [review]
This looks better and makes more sense

File: src/modules/27-platform/auth-settings/components/CreateIPAllowlist/StepOverview/StepOverview.tsx [hunk 1]
```diff
@@ -121,7 +121,15 @@ const StepOverview: React.FC<StepProps<OverviewForm> & StepOverviewProps> = prop
                       inputName: 'name',
                       idName: 'identifier',
                       isIdentifierEditable: !isEdit,
-                      maxInput: 128
+                      maxInput: 128,
+                      inputGroupProps: {
+                        inputGroup: {
+                          onPaste: e => {
+                            formikProps.setFieldValue('name', e.clipboardData.getData('Text'))
```
Comment (ishantmehtaharness) [inline src/modules/27-platform/auth-settings/components/CreateIPAllowlist/StepOverview/StepOverview.tsx:128]: Wr can also pass third argument as true
Then we dont need to call validate method

**Review by ishantmehtaharness — COMMENTED** [review]
Added comment

File: src/modules/27-platform/auth-settings/components/CreateIPAllowlist/StepOverview/StepOverview.tsx (hunk 1 above)
Comment (sahibaHarness) [reply src/modules/27-platform/auth-settings/components/CreateIPAllowlist/StepOverview/StepOverview.tsx:128]: I wasn't aware, creating another PR with this change

**Pull request merged**